config/base.js

```diff
@@ -37,11 +38,16 @@ const SCHEMA_CHILD_PIPELINES = Joi.object()
         startAll: Joi.boolean()
     })
     .unknown(false);
-
+const SCHEMA_SETUP_TEARDOWN_JOB = Job.job.keys({ requires: Joi.forbidden() });
 const SCHEMA_STAGE = Joi.object()
     .keys({
         description: Joi.string(),
-        jobs: Joi.array().items(Job.jobName).min(0)
+        jobs: Joi.array().items(Job.jobName).min(0),
+        setup: SCHEMA_SETUP_TEARDOWN_JOB,
+        teardown: SCHEMA_SETUP_TEARDOWN_JOB,
+        requires: Job.requires,
+        startFrom: Job.jobName,
```

Comment:
Why do we need this field?
I believe stage always starts from `setup` (It is also implicitly created when the setup has not been configured explicitly in screwdriver.yaml).
And jobs that need to be triggered after `setup` is successfully completed should be determined from the workflowGraph.
We could configure to have more than one job to get started in parallel after `setup` is successfully completed.

Reply:
can this handle a scenario like this
<img width="881" alt="Screenshot 2023-05-05 at 11 31 45 AM" src="https://user-images.githubusercontent.com/193126/236539818-999ea73b-bb0e-4d99-bc22-d80e8a41bf5b.png">